.build.yml

```diff
@@ -10,4 +10,4 @@ tasks:
       curl -o site/intro.webm https://l.sr.ht/lJ9C.webm
   - deploy: |
       [ -n "$GITHUB_PR_NUMBER" ] && complete-build
-      tar -cvz -C swaywm.org/site/ . | hut pages publish -d swaywm.org /dev/stdin
+      tar -cvz -C swaywm.org/site/ . | hut pages publish -d swaywm.org
```

Comment:
We should be able to pass directly the directory to hut I think.

Reply:
Done